src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -709,7 +709,7 @@ impl<T: 'static> EventProcessor<T> {
                                 event: MouseInput {
                                     device_id,
                                     state,
-                                    button: Other(x as u8),
+                                    button: Other(x as u16),
```

Comment:
I'm not sure how sane X11 is, but I think it's fine to use u16, given that kernel uses u16 as well.

Reply:
What is the type natively? usize?